### PR TITLE
Feature/deploy note

### PR DESCRIPTION
**Pull request by gibbs-shih**
### CHECK LIST
[Naming convention](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/coding-convention/naming-convention.md) verification: checked
Coding style verification: checked
new Library: 0
new Class / Component: 0
new loop: 1 
/src/crawler.ts (line26) setInterval 5000
high risk: 0
new sql: 0

### UML

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # See the documentation for all the connection string options: https://pris.ly/d/connection-strings
 
 //PostgreSQL
```
Comment (Luphia) [inline .env.sample:7]: - follow [annotation format](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/annotation.md)

File: .env.sample [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
-SECOND_DATABASE_URL="postgresql://name:password@localhost:5432/database_name"
+DATABASE_URL="postgresql://name:password@localhost:5432/database_name"
 
-//SQLite
-DATABASE_URL="file:./dev.db"
\ No newline at end of file
+//network
```
Comment (Luphia) [inline .env.sample:10]: - follow [annotation format](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/annotation.md)

File: src/crawler.ts [hunk 3]
```diff
@@ -20,6 +20,10 @@ class Crawler {
 
   start() {
     this.go();
+    // conduct every 5 seconds
```
Comment (Luphia) [inline src/crawler.ts:23]: - follow [annotation format](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/annotation.md)

**Review by Luphia — CHANGES_REQUESTED** [review]

File: .env.sample (hunk 1 above)
Comment (gibbs-shih) [reply .env.sample]: done~

File: .env.sample (hunk 2 above)
Comment (gibbs-shih) [reply .env.sample]: done~

File: src/crawler.ts (hunk 3 above)
Comment (gibbs-shih) [reply src/crawler.ts]: done~

**Review by Luphia — APPROVED** [review]

**Review by Luphia — APPROVED** [review]
- no bug found

**Pull request merged**